test/spec/api/v4/dataview/formula.spec.js

```diff
@@ -0,0 +1,207 @@
+var Backbone = require('backbone');
+var _ = require('underscore');
+var carto = require('../../../../../src/api/v4/index');
+
+function createFakeInternalModel () {
+  var internalModel = {
```

Comment:
I like to call this kind of objects`internalModelMock` || `fakeInternalModel`, what do you think?

Reply:
internalModelMock